### PR TITLE
faq.md: add 1 FAQ

### PR DESCRIPTION
**Pull request by GMHDBJD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/faq.md [hunk 1]
```diff
@@ -344,3 +344,27 @@ DM-master 会在启动时将 etcd 信息记录在当前目录。如果重启后
 > **注意：**   
 > 
 > 关于 `proxy` 的环境变量有 `http_proxy`，`https_proxy`，`no_proxy` 等。如果依据上述解决方案处理后仍无法连接，可以考虑检查 `http_proxy` 和 `no_proxy` 的参数配置是否有影响。 
+
+## v2.0.2 - v2.0.6 版本执行 start-relay 命令报错该如何处理
```
Comment (Liuxiaozhen12) [inline zh/faq.md:348]:
```suggestion
## v2.0.2 - v2.0.6 版本执行 start-relay 命令报错该如何处理？
```

File: zh/faq.md [hunk 2]
```diff
@@ -344,3 +344,27 @@ DM-master 会在启动时将 etcd 信息记录在当前目录。如果重启后
...
+```
+flush local meta, Rawcause: open relay-dir/xxx.000001/relay.metayyyy: no such file or directory
+```
+
+出现上述错误可能有以下原因：
+
+- DM 从 v2.0.1 及之前的版本升级到 v2.0.2 - v2.0.6 版本，且升级之前曾开启过 relay log，升级完后重新开启。
+- 使用 stop-relay 命令暂停 relay log 后重新开启
+
+上述条件下，有一定概率触发此错误。
```
Comment (Liuxiaozhen12) [inline zh/faq.md:359]:
```suggestion
上述报错在以下情况下有可能会被触发：

- DM 从 v2.0.1 及之前的版本升级到 v2.0.2 - v2.0.6 版本，且升级之前曾开启过 relay log，升级完后重新开启。
- 使用 stop-relay 命令暂停 relay log 后重新开启 relay log。

```

File: en/faq.md [hunk 3]
```diff
@@ -374,4 +374,28 @@ To solve this issue, check whether `https_proxy` is mandatory. If not, cancel th
...
+
+## How to deal with the error of executing start-relay command from v2.0.2 to v2.0.6?
+
+```
+flush local meta, Rawcause: open relay-dir/xxx.000001/relay.metayyyy: no such file or directory
+```
+
+The above error can be caused by the following reason:
+
+- DM has been upgraded from v2.0.1 and earlier to v2.0.2 - v2.0.6, and relay log was started before the upgrade and restarted again after the upgrade
```
Comment (Liuxiaozhen12) [inline en/faq.md:387]:
```suggestion
- DM has been upgraded from v2.0.1 and earlier to v2.0.2 - v2.0.6, and relay log is started before the upgrade and restarted after the upgrade.
```

File: en/faq.md [hunk 4]
```diff
@@ -374,4 +374,28 @@ To solve this issue, check whether `https_proxy` is mandatory. If not, cancel th
...
+- Use the stop-relay command to pause the relay log and then restart it
```
Comment (Liuxiaozhen12) [inline en/faq.md:388]:
```suggestion
- Execute the stop-relay command to pause the relay log and then restart it
```

**Review by lance6716 — COMMENTED** [review]
rest lgtm

File: zh/faq.md [hunk 5]
```diff
@@ -344,3 +344,27 @@ DM-master 会在启动时将 etcd 信息记录在当前目录。如果重启后
...
+可以通过以下方式绕过这个问题：
+
+1. 重启 relay log
+
+    ```
+    » stop-relay -s sourceID workerName
+    » start-relay -s sourceID workerName
+    ```
+
+2. 升级 DM 至 v2.0.7 及之后版本
```
[the remark this answers is not in the document]
Comment (GMHDBJD) [reply zh/faq.md:370]: 不同的

File: zh/faq.md [hunk 6]
```diff
@@ -344,3 +344,27 @@ DM-master 会在启动时将 etcd 信息记录在当前目录。如果重启后
...
+## v2.0.2 - v2.0.6 版本执行 start-relay 命令报错该如何处理
+
+```
+flush local meta, Rawcause: open relay-dir/xxx.000001/relay.metayyyy: no such file or directory
+```
+
+出现上述错误可能有以下原因：
+
+- DM 从 v2.0.1 及之前的版本升级到 v2.0.2 - v2.0.6 版本，且升级之前曾开启过 relay log，升级完后重新开启。
+- 使用 stop-relay 命令暂停 relay log 后重新开启
```
[the remark this answers is not in the document]
Comment (GMHDBJD) [reply zh/faq.md]: 第一次错误，第二次成功🥲

File: en/faq.md [hunk 7]
```diff
@@ -374,4 +374,26 @@ To solve this issue, check whether `https_proxy` is mandatory. If not, cancel th
...
+```
+flush local meta, Rawcause: open relay-dir/xxx.000001/relay.metayyyy: no such file or directory
+```
+
+The above error might be made in the following cases:
+
+- DM has been upgraded from v2.0.1 and earlier to v2.0.2 - v2.0.6, and relay log is started before the upgrade and restarted after the upgrade.
+- Execute the stop-relay command to pause the relay log and then restart it.
+
+You can avoid this error by the following steps:
```
Comment (Liuxiaozhen12) [inline en/faq.md:390]:
```suggestion
You can avoid this error by the following options:
```

File: en/faq.md [hunk 8]
```diff
@@ -374,4 +374,26 @@ To solve this issue, check whether `https_proxy` is mandatory. If not, cancel th
...
+
+1. restart relay log
+
+    ```
+    » stop-relay -s sourceID workerName
+    » start-relay -s sourceID workerName
+    ```
+
+2. upgrade DM to v2.0.7 and later
```
Comment (Liuxiaozhen12) [inline en/faq.md:399]:
```suggestion
- Restart relay log:

    ```
    » stop-relay -s sourceID workerName
    » start-relay -s sourceID workerName
    ```

- Upgrade DM to v2.0.7 or later versions.
```

File: zh/faq.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Liuxiaozhen12) [reply zh/faq.md:368]:
```suggestion
可以通过以下方式解决该问题：

- 重启 relay log:

    ```
    » stop-relay -s sourceID workerName
    » start-relay -s sourceID workerName
    ```

- 升级 DM 至 v2.0.7 或之后版本。
```

**Review by Liuxiaozhen12 — APPROVED** [review]
rest LGTM

**Pull request merged**